### PR TITLE
Add validobj recipe

### PR DESCRIPTION
**Pull request by Zaharid**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Zaharid** [comment]
@conda-forge-admin, please ping team

File: recipes/validobj/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/validobj/meta.yaml]: Can you remove the comments from the example

File: recipes/validobj/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "flit install"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```
Comment (chrisburr) [inline recipes/validobj/meta.yaml]:
```suggestion
```

File: recipes/validobj/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - validobj
+
+about:
+  home: https://github.com/Zaharid/validobj
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Validobj gives you valid objects'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/validobj/meta.yaml]:
```suggestion
```

**Comment by Zaharid** [comment]
Thanks for the speedy review!

**Pull request merged**